packages/workspace/src/workspace/workspace.ts

```diff
@@ -910,6 +927,20 @@ export const loadWorkspace = async (
     ?? []
     return result.filter(values.isDefined).flat()
   }
+  const getChangedElementsBetween = async (
+    after: string,
+    before?: string,
+    envName?: string,
+  ): Promise<ElemID[]> => {
+    const env = envName ?? currentEnv()
+    const workspace = await getWorkspaceState()
+    const beforeTimestamp = before ? Date.parse(before) : Date.now()
+    const afterTimestamp = Date.parse(after)
+    const relevantTimestamps = (await awu(workspace.states[env].changedAt.keys()).toArray())
+      .filter(t => Date.parse(t) <= beforeTimestamp && Date.parse(t) >= afterTimestamp)
```

Comment:
nit
```suggestion
    const relevantTimestamps = await awu(workspace.states[env].changedAt.keys())
      .filter(t => Date.parse(t) <= beforeTimestamp && Date.parse(t) >= afterTimestamp)
      .toArray()
```
Also, please rename `t` to something more informative